geometry_msgs/PoseStamped current_pose # Current position
geometry_msgs/PoseStamped goal_pose # Goal
float32 dist_to_goal # Path distance left to goal (m)
float32 time_to_goal # Time left to goal to navigate along path (s)
